### DISCUSSION TITLE
Can't figure out how to use a localleader

### DISCUSSION THREAD

**Discussion by JPDucky** [nvim-neorg/neorg#1021] [Q&A]
I want to use ',' as my local leader, but I can't figure out set it for Neorg. I'm using LazyVim, and I would assume I would need to change something in `neorg/lua/neorg/modules/core/keybinds/keybinds.lua`, but just changing `local leader = keybinds.leader` to `keybinds.localleader` doesn't work. I didn't think it would, but I figured I'd see if it worked anyway- it didn't.

My eventual goal is to have the Neorg keybinds on the localleader set to `,`, with them getting picked up by `which-key` and setting the various prefixes to account for the various types of Neorg commands.

**Comment by benatouba** [comment]
Don't know if this is solved but the localleader is something you set globally and not plugin-specific. I don't immediately find the best place  to set custom keybindings. 
Quick and dirty option would to add `vim.g.maplocalleader = ","` to the end of your `init.lua`.
Beware that this might interfere with LazyVim updates. Find out where to set custom keybindings in LazyVim for the proper solution in the LazyVim-Distro.
